cs/form-validation.texy

```diff
@@ -69,6 +69,28 @@ JavaScript
 
 Validační pravidla se na stranu JavaScriptu přenášejí v HTML atributech `data-nette-rules`, které obsahují JSON popisující jednotlivá pravidla nebo podmínky. Samotnou validaci pak provádí skript, který odchytí událost `submit`, projde jednotlivé prvky a vykoná příslušnou validaci. Výchozí implementací je soubor `netteForms.js`, který najdete v distribuci v adresáři `src/assets`. Stačí jej tedy do stránky zalinkovat:
```

Comment:
Nenapadá tě, jak upravit tuhle větu? "Stačí jej tedy do stránky zalinkovat:" patřilo k tomu `<script>`, ale v případě ukázek s importy je to potřeba napsat nějak jinak.

Reply:
Lepší? https://github.com/nette/docs/pull/823/commits/fbbe4bc6f0c50330cf8c3adab75fefa1e5e8e13b (Nevím jen, jak se chová texy při novém řádku - ignoruje ho jako v MD, nebo ho bere v potaz?)